Floofbot/Modules/Conversions.cs

```diff
@@ -0,0 +1,165 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Text.RegularExpressions;
+using Discord;
+using Discord.Addons.Interactive;
+using Discord.Commands;
+using UnitsNet;
+
+namespace Floofbot
```

Comment:
Does this need to be an entire module? I think it would be better in utilities

Reply:
Might be worth moving it to utilities, I coded it in a different file for clarity on my end.